client-tutorial/README.md

```diff
@@ -1,15 +1,15 @@
+This project does not depend on any propietary or commercial libraries. The project is using Python 3.9+. The largest library dependecy for this project is for the Linux For Health (https://github.com/LinuxForHealth/connect) repo (which will both provide code as well as containerized runtime services. To utilize the LFH tooling you need to clone the connect repo and have docker (with docker-compose) installed and subsequently go to the connect folder and you will see the docker-compose file in that directory and you want to execute `docker-compose up`. Now depending on your platform you *may* need to launch the docker engine application (like on a macintosh the GUI docker application must be running for docker services to exist (i.e. there is no command docker-compose when the docker app is not running). Handily the desktop engine lets you forever after start your containers from inside the GUI; And like any other docker-compose you can edit the launch config for the service (such as adding other tooling you might want such as MySQL or elastic search **(note using the containerized versions of a database may not be storage backed, as docker by default does not provide system storage to a container)**. Other dependecies you will need are located in python itself such as Kivy (this application uses Kivy to generate a cross-platform GUI). Obviously that is not required and feel free to modify this demo to use your favorite front end (or turn it into a web service and use react or whatever). Kivy was just easy since the project will run as a local desktop application. Given the mysql dependency I chose to use PyMysql and SqlAlchemy (ORM) to handle the database access. If you hate ORMs, you are probably a *psychopath*, but it's a free country. You will also need httpx, the pydantic fhir.resources, and a few other pip includes. You don't need to manually add these if you are not running your application locally (i.e. in an IDE). In PyCharm you can click on the **python packages** button at the bottom and search for whatever package you need and it will automagially add them to your venv).
```

Comment:
I just noticed "Linux For Health" as well.  Should we change that to LinuxForHealth?

Reply:
Good catch @ccorley - yes LinuxForHealth is our current "branding" so to speak.